cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -329,7 +329,7 @@ elseif(ARM OR AARCH64)
       ocv_update(CPU_VFPV3_FLAGS_ON "-mfpu=vfpv3")
       ocv_update(CPU_NEON_FLAGS_ON "-mfpu=neon")
       ocv_update(CPU_NEON_FLAGS_CONFLICT "-mfpu=[^ ]*")
-      ocv_update(CPU_FP16_FLAGS_ON "-mfpu=neon-fp16")
+      ocv_update(CPU_FP16_FLAGS_ON "-mfpu=neon-fp16 -mfp16-format=ieee")
```

Comment:
Please remove lines 620-622 to avoid compiler option duplication.

Reply:
Thank you!
Fixed

Reply:
Hmm....
Now ```-mfp16-format=ieee``` is missing in Android build....

Reply:
It is consistency check from CI side. These checks are updated.